client/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
```

Comment:
Should we also add `.classpath`, `.idea/`, `.settings`, and other idea specific dir / files to the list?

Reply:
sure